src/db/migrations/20230703010356_add-profile-table.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up(knex) {
+  return knex.schema
+    .createTable('profiles', table => {
+      table.increments('id').primary()
+      table.integer('onerep_profile_id').references('subscribers.onerep_profile_id').nullable()
+      table.integer('first_name')
+      table.integer('last_name')
+      table.integer('city_name')
+      table.integer('state_code')
+      table.date('date_of_birth')
+      table.timestamp('created_at').defaultTo(knex.fn.now())
+      table.timestamp('updated_at').defaultTo(knex.fn.now())
+      table.index('onerep_profile_id')
+    })
+}
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function down(knex) {
+  return knex.schema.dropTable('profiles')
```

Comment:
See  https://github.com/mozilla/blurts-server/pull/3157/files#r1255721542 suggestion above about potentially renaming this table from "profiles" to "onerep_profiles".